app/src/main/java/ch/epfl/sweng/swenggolf/ListOfferActivity.java

```diff
@@ -0,0 +1,100 @@
+package ch.epfl.sweng.swenggolf;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListOfferActivity extends Activity {
+    private RecyclerView.Adapter mAdapter;
+    protected static final List<Offer> offerList = new ArrayList<>();
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_list_offer);
+
+        setRecyclerView();
+    }
+
+    private void setRecyclerView() {
+        RecyclerView mRecyclerView = findViewById(R.id.offers_recycler_view);
+        mRecyclerView.setHasFixedSize(true);
```

Comment:
I think we can leave it like that for now, because :

- It makes sense that every offer has the same size, as to not favour any of them. A "bigger" offer could be more visible, which is not fair.
- The description is limited to 140 characters now, so it wouldn't make a great difference.
- As for the pictures, we could just use a default one for offer that have no picture, or just force users to put a picture.

Reply:
It's okay for me.